### PR TITLE
Gv final revision graph

### PR DESCRIPTION
**Pull request by giovannivolpe**
graph / binarize
graph / dediagonalize
graph / semipositivize
graph / standardize
graph / symmetrize
graph / test_binarize
graph / test_dediagonalize
graph / test_semipositivize
graph / test_standardize
graph / test_symmetrize

### CODE REVIEW

File: braph2/graph/binarize.m [hunk 1]
```diff
@@ -0,0 +1,11 @@
+function B = binarize(A, varargin)
+% BINARIZE binarizes a matrix
```
Comment (annacanal) [inline braph2/graph/binarize.m:2]: We are not using the varargin here, so now that we are rewriting braph2, this should be removed.

File: braph2/graph/symmetrize.m [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+switch lower(symmetrize_rule)
+    case 'sum'  % sum rule
+        B = A + transpose(A);
+    case 'average'  % average rule
+        B = (A + transpose(A)) / 2;
+    case 'min'  % minimum rule
+        B = min(A, transpose(A));
+    otherwise  % 'max'  % maximum rule
+        B = max(A, transpose(A));
+
```
Comment (mitemijalkov) [inline braph2/graph/symmetrize.m:27]: Just the new line to be removed to be consistent with switch structures in the other functions.

File: braph2/graph/binarize.m (hunk 1 above)
Comment (giovannivolpe) [reply braph2/graph/binarize.m:2]: it's for consistency

File: braph2/graph/symmetrize.m (hunk 2 above)
Comment (giovannivolpe) [reply braph2/graph/symmetrize.m]: done

**Pull request merged**